lib/models/Player.js

```diff
@@ -328,6 +353,20 @@ Player.prototype.unMute = function unMute() {
     { mute: 0 });
 };
 
+Player.prototype.unMuteGroup = function unMuteGroup() {
+  return soap.invoke(
+      `${this.baseUrl}/MediaRenderer/GroupRenderingControl/Control`,
+      TYPE.GroupMute,
+      { mute: 0 });
+};
+
+Player.prototype.muteGroup = function muteGroup() {
+  return soap.invoke(
+      `${this.baseUrl}/MediaRenderer/GroupRenderingControl/Control`,
+      TYPE.GroupMute,
+      { mute: 1 });
+};
+
```

Comment:
The norm that is used on other methods, is to use a boolean flag to indicate on/off, instead of two different methods. Maybe we should stick to that for conformity.